src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Tests/app.config

```diff
@@ -25,24 +25,41 @@
     <add key="ClientSettingsProvider.ServiceUri" value="" />
 	
     <!-- RP related -->
-    <add key="RsVaultRgNameRP" value="restorerg1" />
-    <add key="RsVaultNameRP" value="restorern1" />
-    <add key="RsVaultIaasVMContainerUniqueNameRP" value="iaasvmcontainerv2;shswain-rg1;shswain-vm1" />
-    <add key="RsVaultIaasVMItemUniqueNameRP" value="iaasvmcontainerv2;shswain-rg1;shswain-vm1" />
+    <add key="RsVaultRgNameRP" value="labRG1" />
+    <add key="RsVaultNameRP" value="idcdlslbRSVault" />
+    <add key="RsVaultIaasVMContainerUniqueNameRP" value="iaasvmcontainerv2;arpittestresourcegroup;vkencryrestore" />
+    <add key="RsVaultIaasVMItemUniqueNameRP" value="iaasvmcontainerv2;arpittestresourcegroup;vkencryrestore" />
     <add key="vaultLocationRP" value="westus" />
-    <add key="RecoveryPointNameRP" value="23864730435904" />    
+    <add key="RecoveryPointNameRP" value="13738976139650" />    
     
 	<!-- Restore related -->
-    <add key="RsVaultRgNameRestore" value="shankcanaryrg" />
-    <add key="RsVaultNameRestore" value="shankcanary2" />
-    <add key="RsVaultIaasVMContainerUniqueNameRestore" value="iaasvmcontainerv2;shraccanrg;shraccanv2" />
-    <add key="RsVaultIaasVMItemUniqueNameRestore" value="iaasvmcontainerv2;shraccanrg;shraccanv2" />
-    <add key="vaultLocation" value="southeastasia" />
-    <add key="RecoveryPointName" value="30641628955971" />
-    <add key="StorageAccountId" value="/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourceGroups/shraccanrg/providers/Microsoft.Storage/storageAccounts/shraccanseastorv2" />
-    <add key="SourceVmIdForRestore" value="/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourceGroups/shraccanrg/providers/Microsoft.Compute/virtualMachines/shraccanv2" />
+    <add key="RsVaultRgNameRestore" value="labRG1" />
```

Comment:
See previous comments. we need to remove this file and create any necessary resources as part of test setup